test/algorithmiaTest.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const assert = this.chai ? this.chai.assert : require('assert');
```

Comment:
Since this is running in the global scope let's replace `this` with `window` to make this a bit more clear.
```suggestion
const assert = window.chai ? window.chai.assert : require('assert');
```